### ISSUE TITLE
[Bug]: tank-mode delving Brann causes AddSpamMessage spam

### ISSUE THREAD

**Issue by Farmbuyer** [dandruff/xCT#262]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### WoW version

11.1.0

### xCT+ version

4.9.21

### What happened?

Tried running a delve with Brann in tank mode for the first time.  One of his abilities is the electric Sentry Totem, which redirects incoming damage to the player onto Brann:

[https://www.wowhead.com/spell=474355/lightning-link](https://www.wowhead.com/spell=474355/lightning-link)

Each time this triggers, xCT prints out:

`AddSpamMessage(): empty message for frame damage mergeId 474355 message -0`

and because the totem pulses often during some fights (incoming AoE, etc), this can result in a boatload of spam text being printed to the main chat window.

I've commented out that specific `self:Print` line in my local installation of xCT, but maybe the `AddSpamMessage` routine could be modified to only print out those lines when xCT itself is put into a "player is trying to adjust the displays, print all the debugging messages" mode, or something similar?

### Related Errors

```lua

```

**Comment by RedAces** [comment]
I've had a fix for this in my local branch but never released it. 
Sorry!
Should be fixed with 4.22!

**Comment by Farmbuyer** [comment]
Cheers, thanks!

(I also realized I mis-named the ability, it's Lightning Link as per the URL, not the Sentry Totem.  Derp.)
